test/isolated/config.test.js

```diff
@@ -85,4 +85,27 @@ describe('config', () => {
     )
     expect(config.target).toBe('serverless')
   })
+
+  it('Should throw  error when next.config.ts is there', () => {
+    try {
+      loadConfig(
+        PHASE_DEVELOPMENT_SERVER,
+        join(__dirname, '_resolvedata', 'typescript-config')
+      )
+      // makes sure we don't just pass if the loadConfig passes while it should fail
+      throw new Error('failed')
+    } catch (err) {
+      expect(err.message).toMatch(
+        /Configuring Next.js via next.config.ts is not supported. Please replace the file with next.config.js. Alternatively, you can compile next.config.ts on each change, but this is not recommended/
+      )
+    }
+  })
+
+  it('Should Ignore next.config.ts when next.config.js and next.config.ts are there', () => {
```

Comment:
```suggestion
  it('should not throw an error when two versions of next.config.js are present', () => {
```

Reply:
> 

should not throw an error when more than one type of next.config.js is present